app/views/layouts/doorkeeper.html.haml

```diff
@@ -5,7 +5,7 @@
     = stylesheet_link_tag 'revised', media: 'all'
     <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,400italic,300italic,700' rel='stylesheet' type='text/css'>
     <link href='https://fonts.googleapis.com/css?family=Montserrat:400,700' rel='stylesheet' type='text/css'>
-    = javascript_include_tag 'application'
+    = javascript_include_tag 'application', skip_pipeline: true
```

Comment:
Why does this need to skip pipeline? Does the pipeline no longer do anything useful? e.g. fingerprinting?

No need to answer this before shipping, just something I'm curious about

Reply:
Good nudge, thanks! The `:skip_pipeline` option was a step in the wrong direction — there's a bug on master here — it should be `javascript_pack_tag` since webpacker is handling that `application.js`. The jQuery source is also missing:

When I visit http://localhost:3001/oauth/applications on master, I see the following in the JS console:

```
GET http://localhost:3001/javascripts/application.js net::ERR_ABORTED 404 (Not Found)
applications:5507 Uncaught ReferenceError: $ is not defined
    at applications:5507
```

The original line raises an `Sprockets::Rails::Helper::AssetNotFound` exception on this branch because of the Sprockets upgrade.

Fixed in 66bd3546